packages/elements/src/components/ui/buttons/menu-button/MenuButtonGroupBox.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import { ReactNode } from "react";
+import styles from "./MenuButtonGroupBox.module.css";
+
+export interface MenuButtonGroupBoxProps {
+  children?: ReactNode;
+}
+
+export const MenuButtonGroupBox: React.FC<MenuButtonGroupBoxProps> = ({
+  children,
+}) => {
+  return <div className={styles.menuButtonGroupBox}>{children}</div>;
```

Comment:
Why not make this a Box? :)

Reply:
It could be :) 
We use `--current-text-size` to control the size, so when a MenuButton is within another MenuButton (that is expanded), the text size is decreased.

I try to do all styling in CSS module, or all styling in JSX, not mixed.
I don't always succeed, but... I try :) 

So that is why, but it could be a Box :)